src/main/features/dashboard/views/index.njk

```diff
@@ -44,25 +44,13 @@
             </tr>
           {% endif %}
           {% for claim in claimsAsClaimant %}
-            {% if claim.eligibleForCCJ %}
```

Comment:
Something to improve in the future. Evaluating status in template is a bad idea because it is difficult to test. We should have class evaluating status, well tested. Then we pass status to the template to match it with the text for end user.

Reply:
I'd leave it as it is now, as this story is taking too long.

This story also doesn't cover all possible statuses as we haven't implement all paths yet (hands off). Once all paths are implemented and we have very final version of statuses and dashboards I'd be more than happy to move it to a proper unit testable class.

Reply:
Definitely not for now, next time possibly when we touch statuses.